float32 pitch
float32 roll
float32 thrust
---
bool success
string message
